typescript/schemas/src/internal/schemas.ts

```diff
@@ -863,28 +863,73 @@ const RawImage: FoxgloveMessageSchema = {
     {
       name: "width",
       type: { type: "primitive", name: "uint32" },
-      description: "Image width",
+      description: "Image width in pixels",
     },
     {
       name: "height",
       type: { type: "primitive", name: "uint32" },
-      description: "Image height",
+      description: "Image heigh in pixels",
     },
     {
       name: "encoding",
       type: { type: "primitive", name: "string" },
       description:
-        "Encoding of the raw image data\n\nSupported values: `8UC1`, `8UC3`, `16UC1` (little endian), `32FC1` (little endian), `bayer_bggr8`, `bayer_gbrg8`, `bayer_grbg8`, `bayer_rggb8`, `bgr8`, `bgra8`, `mono8`, `mono16`, `rgb8`, `rgba8`, `uyvy` or `yuv422`, `yuyv` or `yuv422_yuy2`",
+        "Encoding of the raw image data. See the `data` field description for supported values."
     },
     {
       name: "step",
       type: { type: "primitive", name: "uint32" },
-      description: "Byte length of a single row",
+      description: "Byte length of a single row. This is usually some multiple of `width` depending on the encoding, but can be greater to incorporate padding."
     },
     {
       name: "data",
       type: { type: "primitive", name: "bytes" },
-      description: "Raw image data",
+      description: `Raw image data.
+
+The requirements for each \`encoding\` value are:
+- \`yuv422\`,  \`uyvy\`:
```

Comment:
what order are these in? I'd probably order them either alphabetically, by bit depth/number of channels/etc., or by popularity